src/pages/DatasetCatalog.js

```diff
@@ -615,7 +616,7 @@ class DatasetCatalog extends Component {
               onClick: this.download,
               className: 'col-lg-5 col-md-5 col-sm-5 col-xs-5 btn-primary dataset-background'
             }, [
-              'Download selection',
+              'Download Dataset Info',
```

Comment:
Food for thought from my English-minor days: `Info` is a colloquial term that doesn't connote a professional tone, IMHO.

Reply:
Agreed - changed to "List"